grc/python/flow_graph.tmpl

```diff
@@ -333,13 +333,13 @@ $short_id#slurp
 
 
 def argument_parser():
-    parser = OptionParser(option_class=eng_option, usage="%prog: [options]")
+    parser = ArgumentParser()
     #for $param in $parameters
         #set $type = $param.get_param('type').get_value()
         #if $type
             #silent $params_eq_list.append('%s=options.%s'%($param.get_id(), $param.get_id()))
-    parser.add_option(
-        "$make_short_id($param)", "--$param.get_id().replace('_', '-')", dest="$param.get_id()", type="$type", default=$make_default($type, $param),
+    parser.add_argument(
+        "$make_short_id($param)", "--$param.get_id().replace('_', '-')", dest="$param.get_id()", type=$type, default=$make_default($type, $param),
```

Comment:
not sure this is (or ever was) safe:
```
type=$type
```
From parameter.xml I see values like eng_float or intx for which we don't have handlers here.
I recommend adding a new <opt> in the xml to specify the argparse_type. Read it by changing line 338 to ``` $param.get_param('type').get_option('argparse_type')```